### PR TITLE
Batch of new System and Error/Info screens.

### PR DESCRIPTION
**Pull request by littlenaut**

### CODE REVIEW

**Comment by littlenaut** [comment]
[New machine-ui 'Nothing yet' screen](https://trello.com/c/Sx31ouVX/844-new-machine-ui-nothing-yet-screen)

**Comment by littlenaut** [comment]
[New machine-ui 'Front facing cam' screen](https://trello.com/c/Ruhemt9r/845-new-machine-ui-front-facing-cam-screen)

**Comment by littlenaut** [comment]
[New machine-ui 'Look into the camera' screen](https://trello.com/c/HGiwAFtn/853-new-machine-ui-look-into-the-camera-screen)

**Comment by littlenaut** [comment]
[New machine-ui 'Funds Received' screen](https://trello.com/c/s9DRxiwj/854-new-machine-ui-funds-received-screen)

**Comment by littlenaut** [comment]
[New machine-ui 'Temporary limit reached' screen](https://trello.com/c/fyMx7tki/852-new-machine-ui-temporary-limit-reached-screen)

**Comment by littlenaut** [comment]
[New machine-ui 'Want to try again?' screen](https://trello.com/c/NPUwoU6G/858-new-machine-ui-want-to-try-again-screen)

**Comment by littlenaut** [comment]
[New machine-ui 'We're all out of cash!' screen](https://trello.com/c/8qXKL9G8/874-new-machine-ui-were-all-out-of-cash-screen)

**Comment by littlenaut** [comment]
[New machine-ui 'More, please' screen](https://trello.com/c/stQkI98I/875-new-machine-ui-more-please-screen)

**Comment by littlenaut** [comment]
[New machine-ui 'Network Down' screen](https://trello.com/c/1bhvEGlC/877-new-machine-ui-network-down-screen)

**Comment by littlenaut** [comment]
[New machine-ui 'Waiting' screen](https://trello.com/c/RFIlMFOz/878-new-machine-ui-waiting-screen)

File: ui-new/css/main.scss [hunk 1]
```diff
@@ -86,6 +101,379 @@ footer {
...
+  margin-right: 2.5vw;
+  border-radius: 4vh;
+  padding: 2vh 3vh;
+}
+
+.cash-out-color .mainv .actions .button-yes {
+  border: 2px solid $purple-grey;
+  background: $purple-grey;
+  color: $white;
+  @include museo-bold
```
Comment (RafaelTaranto) [inline ui-new/css/main.scss]: Description-2-bold already includes @museo-bold, was this added for any reason?

File: ui-new/css/main.scss [hunk 2]
```diff
@@ -86,6 +101,379 @@ footer {
...
+.cash-in-color .mainv .actions .button-yes:active,
+.cash-in-color .mainv .actions .button-no:active {
+  border: 2px solid $white;
+  background: $white;
+  color: $sapphire;
+  @include museo-bold
+  @include Description-2-bold
+}
+
+.deposit_network_down_state .mainv .header {
```
Comment (RafaelTaranto) [inline ui-new/css/main.scss]: This part of the css is one of the things I was going to talk to you about. I'm assuming every screen has similar definitions and they are used to correctly place the header icon and h3/h2 on the same position across the screens.

I did the following for similar screens
- Set a header with fixed height
- Set a main with adjustable height
- footer that can be pushed off but with a set height

With this I can create several of these screens on a couple of hours and create a couple of standards based on the different header heights. 

https://github.com/lamassu/lamassu-machine/pull/247/files#diff-fad9caab4b96a0c206ae0c71ef18c0b8R524

File: ui-new/html/deposit-network-down.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<section class="viewport deposit_network_down_state system-color" data-tr-section="deposit-network-down">
+  <section class="mainv">
+    <section class="header">
```
Comment (RafaelTaranto) [inline ui-new/html/deposit-network-down.html]: There is a html `<header>` this is not that important and I'm also using `<section class="main"> ` instead of `<main>` right now.

File: ui-new/css/loading.scss [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+   FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+   COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+   IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+   CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+.sk-three-bounce {
+  margin: 40px auto;
```
Comment (littlenaut) [inline ui-new/css/loading.scss]: Change units from `px` to `vh`/`vw`.

File: ui-new/css/main.scss (hunk 1 above)
Comment (littlenaut) [reply ui-new/css/main.scss]: No, looks like a mistake.

File: ui-new/css/typography.scss [hunk 5]
```diff
@@ -62,28 +62,42 @@
...
 
 @mixin Description-2 {
-  @include mueseo;
+  @include museo;
+  // x800 = 24px
+  font-size: 3vh;
+  line-height: 1.33;
+}
+
+@mixin Description-2-bold {
```
[the remark this answers is not in the document]
Comment (littlenaut) [reply ui-new/css/typography.scss]: In the UI style guide there are two different font weights for `Description-2`: 500 and 700.
That's why I created this one and didn't look for other styles with the same properties. Going to nuke it and replace by `Tagline-2`.

File: ui-new/css/main.scss (hunk 2 above)
Comment (littlenaut) [reply ui-new/css/main.scss]: Yeah, your approach is way better!

File: ui-new/css/loading.scss (hunk 4 above)
Comment (littlenaut) [reply ui-new/css/loading.scss]: Fixed in ba8d197

File: ui-new/css/typography.scss (hunk 5 above)
[the remark this answers is not in the document]
Comment (littlenaut) [reply ui-new/css/typography.scss]: Fixed in 3f4149e

File: ui-new/css/main.scss (hunk 1 above)
Comment (littlenaut) [reply ui-new/css/main.scss]: Fixed in 3f4149e

File: ui-new/html/deposit-timeout.html [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    <section class="header">
+      <img src="images/nothing-yet.svg"/>
+      <h3 class="js-i18n">Nothing yet.</h3>
+    </section>
+    <section class="info">
+      <p class="js-i18n d2">No coins are showing up.</p>
+      <p class="js-i18n d2b">Have you sent the BTC yet?</p>
+    </section>
+    <section class="actions">
+      <span class="button-yes js-i18n" id="deposit-timeout-sent-yes">Yes, I have</span>
```
Comment (RafaelTaranto) [inline ui-new/html/deposit-timeout.html]: These buttons won't work, if you check out app.js there is a function called `targetButton` that travels the dom node looking for specific classes before stopping, you should add a `button` class here and on all buttons

File: ui-new/html/hard-limit-reached.html [hunk 7]
```diff
@@ -0,0 +1,15 @@
+<section class="viewport hard_limit_reached_state cash-in-color" data-tr-section="hard-limit-reached">
```
Comment (RafaelTaranto) [inline ui-new/html/hard-limit-reached.html]: I think this screen should work for both cash-in and cash-out, checkout `setDirection` on app.js on how to do this, the cash-in-color should be removed from this. Also make sure to add it to test-app.js so it doesn't start as a blank page while testing.

File: ui-new/html/hard-limit-reached.html [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+  <section class="mainv">
+    <section class="header">
+      <img src="images/temporary-limit-amazonite.svg"/>
```
Comment (RafaelTaranto) [inline ui-new/html/hard-limit-reached.html]: We should try to add a width to all svgs to make them responsive.

File: ui-new/html/limit-reached.html [hunk 9]
```diff
@@ -1,24 +1,14 @@
...
-  <footer>
-    <button
-      id="limit-reached-ok"
-      display="inline-block"
-      class="filled-progress-button button ok js-i18n"
-    >
-      OK
-    </button>
-  </footer>
+<section class="viewport limit_reached_state cash-out-color" data-tr-section="limit-reached">
```
Comment (RafaelTaranto) [inline ui-new/html/limit-reached.html]: Clearing up something from the chat. The screen that got removed here is still a valid screen called limit_reached_state, the one that got added should be called out_of_cash_state.

File: ui-new/html/minimum-tx.html [hunk 10]
```diff
@@ -0,0 +1,11 @@
+<section class="viewport minimum_tx_state cash-in-color" data-tr-section="minimum-tx">
+  <section class="mainv">
+    <section class="header">
+      <img src="images/more-please.svg"/>
+      <h2 class="js-i18n">More, please</h2>
+    </section>
+    <section class="info">
+      <p id="js-i18n-lowest-bill" class="d1">Minimum first bill is</p>
```
Comment (RafaelTaranto) [inline ui-new/html/minimum-tx.html]: I don't think this one will work either. js-i18n-lowest-bill will get its contents replaced by `Please insert %s or more.`. Look for lowest-bill on app.js and you will find the piece of code that does that.

File: ui-new/start.html [hunk 11]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (RafaelTaranto) [inline ui-new/start.html]: This one is not related to this start.html but to the one that lives inside the html folder. Your commits should be removing the html from that to their own separate file but for some reason they are still present there.

File: ui-new/html/waiting.html [hunk 12]
```diff
@@ -0,0 +1,12 @@
+<section class="viewport waiting_state cash-out-color fiat-side" data-tr-section="waiting">
```
Comment (RafaelTaranto) [inline ui-new/html/waiting.html]: On this case there are two different screens for cash_in and cash_out, the other is called cash_in_waiting_state. They should get merged into one, but at this point it would be faster to just replicate this over there.

File: ui-new/html/deposit-network-down.html [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+      <img src="images/network-down.svg"/>
+      <h3 class="js-i18n">Network down</h3>
+    </section>
+    <section class="info">
+      <p class="d2 js-i18n">Sorry, we are down for the moment.</p>
```
Comment (RafaelTaranto) [inline ui-new/html/deposit-network-down.html]: Just revised the screens and this text looks a lot more to what was in a screen called `trouble_state`, network down has slightly different text

**Comment by RafaelTaranto** [comment]
Ohh, also I don't think we ever came around to adding in the logic to the dispensing screens.

**Comment by RafaelTaranto** [comment]
LGTM!
